addons/mass_mailing/static/src/views/mailing_contact_view_list.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+
+import { ListController } from "@web/views/list/list_controller";
+import { listView } from '@web/views/list/list_view';
+import { registry } from '@web/core/registry';
+import { useService } from "@web/core/utils/hooks";
+
+/**
+ * List view for the <mailing.contact> model.
+ *
+ * Add an import button to open the wizard <mailing.contact.import>. This wizard
+ * allows the user to import contacts line by line.
+ */
+export class MailingContactListController extends ListController {
+    async setup() {
+        super.setup();
+        this.actionService = useService("action");
+    }
+
+    onImport() {
+        const context = this.props.context;
+        const actionParams = { context: context };
+        if (!context.default_mailing_list_ids && context.active_model === 'mailing.list' && context.active_ids) {
+            actionParams.context.default_mailing_list_ids = context.active_ids;
+        }
+        this.actionService.doAction('mass_mailing.mailing_contact_import_action', actionParams);
```

Comment:
I'm not sure this part still works properly.

To test:
- Email Marketing app
- Mailing Lists -> Mailing Lists
- Click on a record (e.g: "Interested in Tree Promotions")
- Click on the "Recipients" stat button
- Click on import

Master:
![image](https://user-images.githubusercontent.com/41282536/187361861-022c1663-077f-4411-939b-53fcea276ca6.png)

Your branch:
![image](https://user-images.githubusercontent.com/41282536/187361891-40a99865-056e-4b49-beb8-f285a769b326.png)

(Did not test on kanban but I'm guessing it also has the issue).